lib/al/Library/Shadow/DepthShadowDrawer.h

```diff
@@ -0,0 +1,114 @@
+void setShadowMaskDropLengthEvenWithTarget(al::ShadowMaskBase*, const char*, const sead::Vector3f&);
+void setShadowMaskDropLengthEvenWithTarget(al::ShadowMaskBase*, al::ShadowMaskBase const*, const sead::Vector3f&);
+void setShadowMaskDropLengthEvenWithDrawCategory(al::LiveActor*, al::ShadowMaskDrawCategory, const al::LiveActor*, const char*);
+void setShadowMaskDropLengthEvenPlaneNormal(const al::LiveActor*, const sead::Vector3f&);
+s32 getDepthShadowMapWidth(const al::LiveActor*, const char*);
+s32 getDepthShadowMapHeight(const al::LiveActor*, const char*);
+void setDepthShadowMapSize(const al::LiveActor*, s32, s32, const char*);
+s32 getDepthShadowMapLength(const al::LiveActor*, const char*);
+void setDepthShadowMapLength(const al::LiveActor*, f32, const char*);
+void setDepthShadowMapLengthFromActorTransFlag(const al::LiveActor*, bool, const char*);
+void setDepthShadowMapBoundingBox(const al::LiveActor*, const sead::Vector3f&, const sead::Vector3f&, const char*);
+void setDepthShadowMapMaskTypeNone(const al::LiveActor*, const char*);
+void setDepthShadowMapMaskTypeSelf(const al::LiveActor*, const char*);
+void setEnableDepthShadowMapBottomGradation(const al::LiveActor*, const char*, bool);
+bool isEnableDepthShadowMapBottomGradation(const al::LiveActor*, const char*);
+void setDepthShadowMapBottomGradationLength(const al::LiveActor*, const char*, f32);
+f32 getDepthShadowMapBottomGradationLength(const al::LiveActor*, const char*);
+bool isAppendSubActorDepthShadowMap(const al::LiveActor*);
+f32 getShadowMaskDropLengthMax(const al::LiveActor*);
+void setShadowMaskColor(const al::LiveActor*, const char*, const sead::Color4f&);
+sead::Color4f& getShadowMaskColor(const al::LiveActor*, const char*);
+void setShadowMaskIntensity(const al::LiveActor*, const char*, f32);
+f32 getShadowMaskIntensity(const al::LiveActor*, const char*);
+void* getShadowMaskTextureFixedScale(const al::LiveActor*, const char*);
+void setShadowMaskTextureFixedScale(const al::LiveActor*, const char*, f32);
+sead::Vector3f& getShadowMaskOffset(const al::LiveActor*, const char*);
+void setShadowMaskOffset(const al::LiveActor*, const sead::Vector3f&, const char*);
+bool isExistOcclusionLightPosPtr(const al::LiveActor*, const char*);
+void setOcclusionLightPosPtr(const al::LiveActor*, const char*, const sead::Vector3f*);
+void setOcclusionAddOffset(const al::LiveActor*, const char*, const sead::Vector3f&);
+void enableOcclusion(const al::LiveActor*, const char*);
+void disableOcclusion(const al::LiveActor*, const char*);
+bool isEnableOcclusion(const al::LiveActor*, const char*);
+void calcSphereDoBoundingInfo(sead::Vector3f*, sead::Vector3f*, f32*, f32*, const al::LiveActor*, const char*, f32);
+void calcOcclusionSpherePos(sead::Vector3f*, const al::LiveActor*, const char*);
+void changeOcclusionGroup(const al::LiveActor*, const char*, const char*);
+void* getOcclusionSphere(const al::LiveActor*, const char*);
```

Comment:
wrong return type

Reply:
What would the return type be on this one?

Reply:
`al::OccSphere*` or `const al::OccSphere&`